### DISCUSSION TITLE
InvalidData, error: "stream did not contail valid UTF-8"

### DISCUSSION THREAD

**Discussion by aaronj1234** [thirtythreeforty/neolink#131] [Q&A] [answered]
Hello, 

I was trying to run Neolink today, when I got this error:

![image](https://user-images.githubusercontent.com/77698398/107124833-f38a1880-689d-11eb-916e-89880d9c6cc2.png)

I'm not sure how to fix this.

I copied the Neolink file to the `/usr/local/bin` directory, and did all these commands to try to fix it:

`cd ~/Downloads/`
`cp /usr/local/bin/neolink my_config.toml`
`sudo cp neolink /usr/local/bin/neolink`
`sudo chmod +x /usr/local/bin/neolink`
`neolink --config my_config.toml`

But that still did not fix the problem.

**Comment by QuantumEntangledAndy** [comment, marked as answer]
This line here is wrong

```
cp /usr/local/bin/neolink my_config.toml
```

What you did here was copy the binary file into the config toml.

You need to make the config toml yourself in a text editor. 

Delete your current my_config.toml

Go and find the example config on GitHub and download that. https://github.com/thirtythreeforty/neolink/blob/master/sample_config.toml

Edit the file in a text editor to include your cameras details. I recommend notepad++ but any basic texteditor should work do not ms word. 

Rename the example config to `my_config.toml`

Run neolink with the config

```
neolink --config my_config.toml
```

**Reply by aaronj1234** [reply]
Thank you so much!
It's now working very well with my Reolink Lumus 👍👍👍
